debezium-schema-generator/src/main/java/io/debezium/schemagenerator/SchemaGenerator.java

```diff
@@ -12,43 +12,68 @@
 import java.util.Optional;
 import java.util.ServiceLoader;
 import java.util.ServiceLoader.Provider;
+import java.util.logging.Logger;
```

Comment:
I was thinking of `java.lang.System.Log` actually.

Reply:
Changed it!
I thought java.util.logging is the more powerful API falling back to System.Log? Or is it the other way around? These new APIs confuse me.